cime_config/config_grids.xml

```diff
@@ -5660,7 +5660,7 @@
     </gridmap>
 
     <gridmap ocn_grid="SOwISC12to30E3r3" rof_grid="r05">
-      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r05_to_SOwISC12to30E3r3_cstmnn.r150e300.20240808.nc</map>
+      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r05_to_SOwISC12to30E3r3_Ratio0.5_maxFlux0.001.Greenland100x+Antarctica100x.nc</map>
```

Comment:
This doesn't have the right naming convention.  Also make sure its not in netCDF4 format.

Reply:
@rljacob what netcdf format should it be?  using ncks, it seems the possible choices are:

-5, --5, 64bit_data     Output file in netCDF3 64-bit data (aka CDF5, PnetCDF) storage format
-6, --6, 64, 64bit_offset       Output file in netCDF3 64-bit offset storage format

Reply:
-5, --5, 64bit_data.  how about map_r05_to_SOwISC12to30E3r3_Ratio0.5_maxFlux0.001.Greenland100x+Antarctica100x.cstmnn.20241120.nc

Reply:
Or map_r05_to_SOwISC12to30E3r3_rat0.5_maxFlx0.001.Grlnd100x+Ant100x.cstmnn.20241120.nc

Reply:
we want the algorithm and datestring on the end.

Reply:
also remove the "+" sign.  those can lead to trouble in strings.

Reply:
Changed name to `map_r05_to_SOwISC12to30E3r3_rat0.5_maxFlx0.001.Grlnd100x_Ant100x.cstmnn.20241120.nc`.
Output of `ncdump -k` is `64-bit offset`.